exampleSite/content/index/copyright.md

```diff
@@ -0,0 +1,21 @@
++++
+fragment = "copyright"
+#disabled = true
+date = "2016-09-07"
+lastmod = "2017-09-07"
+weight = 1250
+#background = ""
+
+copyright = "" # default: Copyright $Year $params.name
```

Comment:
$param.name is not descriptive here. Probably should be called Site Name or something.

Reply:
Fixed in 5cb5f94